packages/todos-lib/package.js

```diff
@@ -70,4 +70,19 @@ Package.onUse(function(api) {
   api.imply([
     'method',
   ]);
+
+  // i18n
+  api.imply([
+    'tap:i18n@1.7.0'
+  ]);
+
+  // App libs
+  api.use([
+    // TODO -- I need to add this because of a bug in tap:18n I guess?
```

Comment:
Whoa this is intense. So perhaps we should include tap:i18n in the app globally somehow? Then we don't need all of this crazy plumbing, it looks like it wasn't intended to be used in this way.